### PR TITLE
Add view cube

### PR DESCRIPTION
**Pull request by slimbuck**
This PR:
* adds a new "View Cube" widget for fast snapping of the camera to world axes
* bumps package version

Screenshot 2024-07-18 at 14 52 40

### CODE REVIEW

File: src/ui/view-cube.ts [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+
+        shapes.px.children[0].addEventListener('pointerdown', () => { events.fire('camera.align', 'px'); });
+        shapes.py.children[0].addEventListener('pointerdown', () => { events.fire('camera.align', 'py'); });
+        shapes.pz.children[0].addEventListener('pointerdown', () => { events.fire('camera.align', 'pz'); });
+        shapes.nx.children[0].addEventListener('pointerdown', () => { events.fire('camera.align', 'nx'); });
+        shapes.ny.children[0].addEventListener('pointerdown', () => { events.fire('camera.align', 'ny'); });
+        shapes.nz.children[0].addEventListener('pointerdown', () => { events.fire('camera.align', 'nz'); });
+
+        events.on('camera.align', (axis: string) => {
+            console.log(axis);
```
Comment (willeastcott) [inline src/ui/view-cube.ts:93]: Delete?

File: src/editor.ts [hunk 2]
```diff
@@ -160,6 +160,18 @@ const registerEditorEvents = (events: Events, editHistory: EditHistory, scene: S
...
 
+    // handle camera align events
+    events.on('camera.align', (axis: string) => {
+        switch (axis) {
+            case 'px': scene.camera.setAzimElev(90, 0); break;
+            case 'py': scene.camera.setAzimElev(0, -90); break;
+            case 'pz': scene.camera.setAzimElev(0, 0); break;
+            case 'nx': scene.camera.setAzimElev(270, 0); break;
+            case 'ny': scene.camera.setAzimElev(0, 90); break;
+            case 'nz': scene.camera.setAzimElev(180, 0); break;
```
Comment (willeastcott) [inline src/editor.ts:171]: Isn't this crying out for a sexy lerp?

**Review by willeastcott — APPROVED** [review]
I never would have considered writing this as a PCUI element. Very cool!

File: src/editor.ts (hunk 2 above)
Comment (slimbuck) [reply src/editor.ts:171]: It lerps internally :)

File: src/ui/view-cube.ts (hunk 1 above)
Comment (slimbuck) [reply src/ui/view-cube.ts:93]: yes! thanks

**Pull request merged**